azurerm/resource_arm_loadbalancer_test.go

```diff
@@ -209,6 +228,27 @@ resource "azurerm_lb" "test" {
 }`, rInt, location, rInt)
 }
 
+func testAccAzureRMLoadBalancer_standard(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+    name = "acctestrg-%d"
+    location = "%s"
+}
+
+resource "azurerm_lb" "test" {
+    name = "arm-test-loadbalancer-%d"
+    location = "${azurerm_resource_group.test.location}"
+		resource_group_name = "${azurerm_resource_group.test.name}"
```

Comment:
got some mixed tabs and spaces here

Reply:
@paultyng I'm noticing this across multiple files in the provider... are tabs or spaces preferred?